README.md

```diff
@@ -22,7 +25,7 @@ either version 2 of the License, or (at your option) any later version.
 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details,
-which you should have received along with this program (filename: COPYING).
+which you should have received along with this program (filename: elestiaCOPYING).
```

Comment:
please remove garbage

Reply:
File name reset back to the correct file name found in the repository.